test/distributed/_composable/test_compose.py

```diff
@@ -302,6 +302,8 @@ def test_state_dict_fsdp_submodules(self):
                 self.assertIsInstance(tensor, ShardedTensor)
             elif "u2" in fqn:
                 self.assertIsInstance(tensor, torch.Tensor)
+        # Ensure that get_state_dict_type can still correctly get the settings.
+        _ = FSDP.get_state_dict_type(model)
```

Comment:
Is this meant to have some assert on the return value?

Reply:
`get_state_dict_type` will assert if the state_dict_config is not restored.